.github/workflows/deploy-production.yml

```diff
@@ -142,13 +150,53 @@ jobs:
       IHE_GW_KEYSTORE_STOREPASS: ${{ secrets.IHE_GW_KEYSTORE_STOREPASS }}
       IHE_GW_KEYSTORE_KEYPASS: ${{ secrets.IHE_GW_KEYSTORE_KEYPASS }}
 
+  ihe-gw-server:
+    if: needs.files-changed.outputs.ihe-gw-server == 'true'
+    needs: files-changed
+    uses: ./.github/workflows/_deploy-ihe-gw.yml
+    with:
+      deploy_env: "production"
+      ECR_REPO_URI: ${{ vars.ECR_REPO_URI_PRODUCTION }}
+      ECS_CLUSTER: ${{ vars.ECS_CLUSTER_PRODUCTION }}
+      IHE_INBOUND_ECS_SERVICE: ${{ vars.IHE_INBOUND_ECS_SERVICE_PRODUCTION }}
+      IHE_OUTBOUND_ECS_SERVICE: ${{ vars.IHE_OUTBOUND_ECS_SERVICE_PRODUCTION }}
+      AWS_REGION: ${{ vars.API_REGION_PRODUCTION }}
+      IHE_GW_ARTIFACT_URL: ${{ vars.IHE_GW_ARTIFACT_URL }}
+      IHE_GW_KEYSTORE_NAME: ${{ vars.IHE_GW_KEYSTORE_NAME }}
+      IHE_GW_ZULUKEY: ${{ vars.IHE_GW_ZULUKEY }}
+      IHE_GW_CONFIG_BUCKET_NAME: ${{ vars.IHE_GW_CONFIG_BUCKET_NAME }}
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+      IHE_GW_KEYSTORE_STOREPASS: ${{ secrets.IHE_GW_KEYSTORE_STOREPASS }}
+      IHE_GW_KEYSTORE_KEYPASS: ${{ secrets.IHE_GW_KEYSTORE_KEYPASS }}
+
+  # ihe-gw-config:
```

Comment:
assuming this will be added back after the reverts in the TODOs?

Reply:
This is following the same pattern as staging, we still don't have CI/CD for the IHE GW config. Need to expose an endpoint on IHE API GW to GH Actions can call it to update the configs.